index.html

```diff
@@ -1790,19 +1793,19 @@
           When an <a>NFC device</a> comes in communication range,
           <ul>
             <li>
-              If <var>options.target</var> has the value <code>"tag"</code>,
+              If <var>target</var> has the value <code>"tag"</code>,
```

Comment:
@zolkis is right for the current version of the document, since [step 9](https://rawgit.com/zolkis/web-nfc/2c04fe90c72064cb912a5f82329db1fa2033bf39/index.html#dfn-sending-nfc-content) introduces a local variable named _target_.

If it weren't a local variable, I've been using the convention `<code><var>options</var>.target</code>` to pull out members of dictionaries.